lewis/adapters/epics.py

```diff
@@ -218,26 +286,46 @@ def __init__(self, device, arguments=None):
         super(EpicsAdapter, self).__init__(device, arguments)
 
         self._options = self._parse_arguments(arguments or [])
-
-        self._create_properties(self.pvs.values())
+        self._bound_pvs = self._bind_properties(self.pvs)
 
         self._server = None
         self._driver = None
 
+    def _bind_properties(self, pvs):
```

Comment:
This would benefit from a docstring, particularly about the format of `pvs` and the return value.